### ISSUE TITLE
Deprecation warnings in Rails 4

### ISSUE THREAD

**Issue by carlosramireziii** [stanislaw/simple_roles#6]
Using `simple_roles` in a Rails 4 project yields the following deprecation warning when using the `:one` strategy:

> DEPRECATION WARNING: Using #scope without passing a callable object is deprecated. For example `scope :red, where(color: 'red')` should be changed to `scope :red, -> { where(color: 'red') }`. There are numerous gotchas in the former usage and it makes the implementation more complicated and buggy. (If you prefer, you can just define a class method named `self.red`.).

The offending line of code is in lib/simple_roles/one/roles_methods.rb in the `#register_dynamic_methods` method:

```
scope :"#{r}s", where(:role => r.to_s)
```

I believe the fix is simply to change it to:

```
scope :"#{r}s", -> { where(:role => r.to_s) }
```

-OR- to make it a class method instead of a scope:

```
class_eval %{
  def self.#{r}s
    where(:role => r.to_s)
  end
```

**Comment by stanislaw** [comment]
Thanks for report! I will soon release a version fixing this.

**Comment by stanislaw** [comment]
I've just released 0.0.12.

Please, let me know, if it works for you.

**Comment by carlosramireziii** [comment]
Awesome! Yes the deprecation notices are gone and the functionality seems to be in tact. Appreciate the fast turnaround on this.
